src/scss/includes/components/itemList.scss

```diff
@@ -1,42 +1,31 @@
 .itemList-item {
   background: $background;
-
-  &:not(:last-child):after {
-    display: block;
-    content: '';
-    height: 1px;
-    margin: 0 30px;
-    background-color: $separator_color;
-  }
 }
 
 .itemList--hover .itemList-item {
   position: relative;
 
   &:hover {
     background: $surface;
-
-    &:before {
-      content: '';
-      position: absolute;
-      width: 4px;
-      height: 100%;
-      background-color: #FF3B4A;
-    }
   }
 }
 
 @media (max-width: 640px) {
+  .panel--pj .itemList {
```

Comment:
Same idea as my comment on Panel. We are in the css file of a generic component, so you can think of it as an external lib, not editable. These specific rules should be in the only place they apply, meaning `categoryPanel`.

It's ok to target generic selectors in the specific context which use them, but the opposite isn't true (and I know some other files of the project break this rule, but we are making things better step by step :))